packer/ubuntu-14.04_amd64.json

```diff
@@ -86,7 +82,7 @@
       "type": "vagrant-cloud",
       "box_tag": "capgemini/apollo-mesos",
       "access_token": "{{user `access_token`}}",
-      "version": "{{user `version`}}"
+      "version": "{{timestamp}}"
```

Comment:
This affects the Vagrantfile - we need to test that - and maybe tweak the ```vagrant.box_version``` param (or remove it)